tests/integration/test_charm.py

```diff
@@ -30,3 +33,26 @@ async def test_build_and_deploy(ops_test: OpsTest, request):
         ops_test.model.deploy(charm, resources=resources, application_name=APP_NAME),
         ops_test.model.wait_for_idle(apps=[APP_NAME], status="active", timeout=1000),
     )
+
+
+@pytest.mark.abort_on_fail
+async def test_given_loki(ops_test: OpsTest):
+    """Deploy loki and prometheus, and make sure all applications are active."""
+    deploy_prometheus = ops_test.model.deploy(
+        "prometheus-k8s",
+        application_name=PROMETHEUS_APPLICATION_NAME,
+        trust=True,
+    )
+    deploy_loki = ops_test.model.deploy(
+        "loki-k8s", application_name=LOKI_APPLICATION_NAME, trust=True, channel="stable"
+    )
+
+    await asyncio.gather(
+        deploy_loki,
+        deploy_prometheus,
+        ops_test.model.wait_for_idle(
+            apps=[APP_NAME, PROMETHEUS_APPLICATION_NAME, LOKI_APPLICATION_NAME],
+            status="active",
+            timeout=1000,
+        ),
+    )
```

Comment:
I don't really understand this test, shouldn't we integrate loki/prometheus with gocert? Plus the test name is bad

Reply:
oops, i forgot about this